student-library/src/main/java/tds/student/services/abstractions/AbstractConfigLoader.java

```diff
@@ -29,6 +29,7 @@
   protected AppExterns                    _externs;
   protected PTSetup                       _ptSetup;
   protected List<TesteeAttributeMetadata> _testeeAttributeMetadata;
+  protected boolean                       _isSecureBrowserRequired;
```

Comment:
Did you consider using an AtomicBoolean instead since the place that sets this requires synchornization?

Reply:
Good thought.  I haven't looked at that type yet, I'll check it out.